Gopkg.toml

```diff
@@ -76,4 +76,8 @@ ignored = ["github.com/fnproject/fn/cli",
 
 [[override]]
   name = "github.com/docker/docker"
-  revision = "29fc64b590badcb1c3f5beff7563ffd31eb58974"
+  revision = "763d8392612942ff5c32a35f8bdafd7ae93d3321"
```

Comment:
does this point to a specific release? it's generally inadvisable to point to tip / random commit on master for docker

Reply:
(the commit appears to be from 11/17, but merged 8/18, but i didn't find it attached to anything, so i'm asking)

Reply:
thank you